testing_data/model_config.json

```diff
@@ -1,57 +1,144 @@
 {
 	"command": "eval",
 	"gpu": 1,
-        "target_suffix": "_lesion-manual",
-        "roi_suffix": "_seg-manual",
-	"bids_path": "../duke/sct_testing/large/",
+	"log_directory": "testing_script",
+	"debugging": false,
+    "object_detection_params": {
+        "object_detection_path": null
+    },
```

Comment:
Be careful... if the PEP8 police comes around....

Reply:
done in 8f35943